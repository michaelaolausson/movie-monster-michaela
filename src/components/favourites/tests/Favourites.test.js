import React from "react";
import "@testing-library/jest-dom/extend-expect";
import { shallow } from "enzyme";
import Favourites from "./../Favourites";


describe("<Favourites />", () => { 
    const wrapper = shallow(<Favourites />);
    it("should match the snapshot", () => {
        expect(wrapper.html()).toMatchSnapshot();
    });
})