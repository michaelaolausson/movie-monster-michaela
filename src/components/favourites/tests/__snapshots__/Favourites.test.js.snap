// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Favourites /> should match the snapshot 1`] = `"<div class=\\"container\\"><div class=\\"row\\"><div class=\\"col\\"><h2>Favourites <span role=\\"img\\" class=\\"float-right\\" aria-label=\\"red-heart-emoji\\">❤️</span></h2></div></div><div class=\\"row\\"><div class=\\"col\\"><p class=\\"text-muted\\">It&#x27;s empty here!<span role=\\"img\\" aria-label=\\"sad-emoji\\">😥</span></p><p class=\\"text-muted\\">Find favourites to add using our great<a href=\\"/search\\">search</a><span role=\\"img\\" aria-label=\\"heart-eyes-emoji\\">😍</span></p></div></div></div>"`;
